README.md

```diff
@@ -52,15 +52,15 @@ Copy the settings config file for the reducer, 'config/settings-reducer.yaml.tem
 Make sure that you can access the Reducer UI at https://localhost:8090. 
 
 ````bash 
-docker-compose -f config/reducer.yaml -f config/private-network.yaml up 
+docker-compose -f config/settings-reducer.yaml -f config/private-network.yaml up 
 ````
 
 4. Start a combiner  
 
 Copy the settings config file for the reducer, 'config/settings-combiner.yaml.template' to 'config/settings-combiner.yaml'. You do not need to make any changes to this file to run the sandbox. To start the combiner service and attach it to the reducer:
 
 ````bash 
-docker-compose -f config/combiner.yaml -f config/private-network.yaml up 
+docker-compose -f config/settings-combiner.yaml -f config/private-network.yaml up 
```

Comment:
This is not correct? settings-combiner.yaml is used by fedn not docker-compose

Reply:
Omg, yes. Closing.